src/Controller/Backend/Async/AsyncControllerInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Controller\Backend\Async;
+
+interface AsyncControllerInterface
```

Comment:
Describe in comment purpose of this empty interface. Eg. Async Controllers have to implement this interface to use proper widgets for that zone.

Reply:
maybe `AsyncZone` would be a better name for this interface?

Reply:
Done